Taskfile.yml

```diff
@@ -253,6 +254,14 @@ tasks:
       - npm run build
       - npm cit
 
+  test-prql-elixir:
+    # These are mostly copied from the tasks in `test-all.yaml`
+    dir: prql-elixir
+    cmds:
+      - mix deps.get
+      - mix compile
+      - mix test
```

Comment:
Re the tests, these are great too! Thanks for adding them. We can add this `task: test-prql-elixir` to this file's `test-all` at some point

Reply:
Do we need some other test cases also?

Reply:
These work for me locally! Very nice

Reply:
I'm not sure re other test cases. If you're comfortable with the tests like this to start, then that's fine.

Probably we should add the current status to the Readme — i.e. "It's new, has minimal tests, and doesn't yet publish to Hex". That would let us merge now without writing lots of tests, and then we can add more robustness & publishing in the next PRs?

Reply:
(but, in case I wasn't clear — we do still need to add the tests to the GitHub Actions path — otherwise they don't run in CI, which is important)

Reply:
> I'm not sure re other test cases. If you're comfortable with the tests like this to start, then that's fine.
This has some other hidden tests, in elixir we have doc tests which are founded in documentation comments(so we already have 6 tests)

>(but, in case I wasn't clear — we do still need to add the tests to the GitHub Actions path — otherwise they don't run in CI, which is important)

I think i ve accidentally added it to the `Taskfile.yml`, it seems running the tests. I will add to github workflow also